vms/avm/network/network_test.go

```diff
@@ -208,32 +259,19 @@ func TestNetworkIssueTxFromRPC(t *testing.T) {
 func TestNetworkIssueTxFromRPCWithoutVerification(t *testing.T) {
 	type test struct {
 		name          string
-		mempoolFunc   func(*gomock.Controller) xmempool.Mempool
+		mempool       xmempool.Mempool
 		appSenderFunc func(*gomock.Controller) common.AppSender
 		expectedErr   error
 	}
 
 	tests := []test{
-		{
-			name: "can't add transaction to mempool",
```

Comment:
Similarly this test I also don't know if we want to keep... if we wanted to keep what this was intended to test we would probably want to add tests for the same mempool invariants described earlier